### PR TITLE
Implemented Issue 51

### PR DESCRIPTION
**Pull request by campellcl**
Implemented Issue #51. Didn't write a test case because I figured if we wanted to test for equality of a random number filled object we ought to pass it a reference to the number generator itself, or at least a seed to the generator. That would require modifying the method signature so I figured I would run it by you guys first. Seems a little trivial to test. Estimated 30 minutes for issue #51, took 15 min.

### CODE REVIEW

File: junit/MatrixTest.java [hunk 1]
```diff
@@ -437,6 +437,16 @@ public void testGetException()
...
     }
+    /**
+     * testRandom -Tests the generation of a random matrix
+     *  of the specified dimensions. 
+     */
+    @Test
+    public void testRandom() 
+    {
+        //TODO: How to seed random number gen across class instances?
+        Random rand = new Random();
```
Comment (whereswaldon) [inline junit/MatrixTest.java]: We could add a static class member that hold the RNG and have getter/setter methods to access it. That way, we could set its seed before creating the matrix and predict its output. That being said, I'm not sure of the "worth-it" factor of doing that. Thoughts @bclinthall ?

**Review by whereswaldon — COMMENTED** [review]
I see why you didn't want to write the test, but I don't want to approve this without a second opinion.

**Comment by bclinthall** [comment]
@whereswaldon @ccampell
Let's do two tests. One makes sure it gets the dimensions right. The other
makes two 20x30 matrices and makes calls assertNotEquals on their arrays.
Sound good?

**Comment by whereswaldon** [comment]
I'm going to merge this because I've only implemented changes that we've discussed.

**Pull request merged**